### DISCUSSION TITLE
Data Graph x-Achse Jahreszahlen

### DISCUSSION THREAD

**Discussion by ktreut** [jomjol/AI-on-the-edge-device#2552] [Q&A]
Nach dem Update auf 15.3.0 zeigt die X-Achse Jahreszahlen statt wie vorher Uhrzeiten über den zurückliegenden 24Stunden Zeitraum. Übersehe ich irgendeine Einstellung?

**Comment by ktreut** [comment]
Mmhh... Jetzt ist die X-Achse wieder korrekt mit den Uhrzeiten beschriftet. Habe  nichts geändert. Vielleicht war ich nach dem Update zu ungeduldig?

**Reply by caco3** [reply]
Das Graphing-Framework wählt die Datumsdarstellung automatisch. Wenn noch kein oder nur ein Wert da ist, kommt ev. etwas Krummes heraus.
